containers/server-bundle/Dockerfile.nodejava.amd64

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8-jdk-slim
+FROM openjdk:8-jdk-slim as builder
```

Comment:
We don't need to build java, maybe jre is good enough? Or even use alpine.
Size comparison:
- openjdk:8-jre-alpine 84.9M
- openjdk:8-jre-slim 187M
- openjdk:8-jdk-slim 288M

Reply:
i didn't know openjdk had an alpine base. yes, i bet we can move to alpine then. and jre too. if people want to use our docker image for development, its possible to install jdk afterward.